.eslintrc.json

```diff
@@ -5,7 +5,8 @@
         "jquery": true
     },
     "extends": [
-        "airbnb-base"
+        "airbnb-base",
+        "eslint:recommended"
```

Comment:
Is this really necessary on top of `airbnb-base` ?  
From what I'm seeing, all it adds is linewrapping the parameters in `callAPI` function calls, which I'm not really a fan of.

Reply:
The [function-paren-newline](https://eslint.org/docs/rules/function-paren-newline) seems part of [airbnb-base](https://github.com/airbnb/javascript#functions--signature-invocation-indentation).
[function-call-argument-newline](https://eslint.org/docs/rules/function-call-argument-newline) also has something to do with it.

"eslint:recommended" has nothing to do with it.

Do you prefer to turn it on or off?